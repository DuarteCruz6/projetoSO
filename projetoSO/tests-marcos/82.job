WRITE [(q1,83)(l3,71)(x3,49)(v1,8)]
WAIT 2000
DELETE [x3,v1]
WAIT 3000
WRITE [(y8,10)(e8,86)]
BACKUP
WRITE [(g9,83)(u9,36)(z7,16)]
BACKUP
READ [u9]
WRITE [(p0,31)(u8,28)(m6,4)(m7,28)]
READ [l3]
WAIT 3000
SHOW
READ [p0]
DELETE [q1,m6,l3,p0]
WAIT 2000
READ [e8]
WRITE [(z9,75)(o6,3)(t0,32)(l4,5)(l1,77)]
READ [z7]
SHOW
WRITE [(b7,25)]
WRITE [(p3,87)(z4,91)(z2,69)(p2,24)(u1,78)]
READ [z7]
READ [l1]
WRITE [(t3,27)(c8,94)(t4,68)(q6,23)(r1,48)]
BACKUP
SHOW
DELETE [u9,z9,q6,y8,l4]
DELETE [l1,b7,g9]
WRITE [(i6,9)(w4,85)(v8,57)]
WRITE [(z0,39)(f7,32)(k9,29)(x7,24)]
READ [u1]
WRITE [(g8,52)(x4,3)(s2,98)(c6,86)(q2,93)]
READ [e8]
WRITE [(g4,66)(a0,42)(n6,100)]
WRITE [(p7,95)(c9,41)(q1,37)(n0,77)(u6,60)]
WRITE [(h5,1)(j4,84)(f4,69)(m8,63)(f1,25)]
DELETE [e8,f4,m7,u6]
WRITE [(s7,63)(c7,81)]
WRITE [(b7,66)(v5,97)]
WRITE [(r4,12)(s1,53)(u7,61)(u0,88)(s3,68)]
WRITE [(x5,30)(f8,75)]
WRITE [(t1,21)(s6,88)(h3,3)(w1,8)]
WRITE [(i4,9)(b1,67)(q3,87)(k8,50)]
DELETE [s6,g4,x7]
READ [i4]
WRITE [(j9,34)(h6,41)(k7,92)(w6,59)]
WRITE [(d7,64)(h0,89)(y5,62)(z3,45)]
READ [j4]
WRITE [(w2,63)(b9,21)]
READ [i4]
READ [r1]
WRITE [(k0,35)(i0,13)(l5,45)(y4,52)(e1,47)]
WRITE [(h9,75)(h4,43)(e4,21)(l0,9)(w9,79)]
WRITE [(p5,21)(e3,18)(k3,11)(h8,47)(s9,79)]
READ [t0]
READ [w2]
DELETE [y4,q2]
WAIT 3000
READ [s3]
DELETE [v5,w4,y5,f8,k3]
SHOW
WRITE [(u4,86)]
WRITE [(t9,69)(x8,72)(v6,6)]
WRITE [(f6,95)]
WRITE [(x7,22)(q0,20)]
DELETE [f1,k0,l5]
READ [u0]
DELETE [u7,k7,n0,f6,z0]
WRITE [(z0,31)(c2,80)]
READ [l0]
READ [u0]
WAIT 1000
WRITE [(o9,40)(n4,6)(q6,63)]
WRITE [(x1,4)(v1,67)(d6,26)]
READ [a0]
READ [h0]
DELETE [s7,e1,q1]
READ [d6]
WRITE [(a3,62)(l8,28)(b3,96)(n8,86)]
WRITE [(f5,33)]
DELETE [n6]
