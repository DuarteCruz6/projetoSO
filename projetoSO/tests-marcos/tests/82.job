SHOW
WRITE [(j5,64)(i0,62)(z4,31)]
WRITE [(i3,44)(m8,42)(r5,7)(e5,24)(m1,16)]
WRITE [(j0,34)(n7,33)(j3,76)]
WRITE [(c2,72)(e0,90)(y0,53)(y0,69)(l2,42)]
WRITE [(h8,21)(f8,85)(k6,85)]
WRITE [(n3,29)(n4,86)(c1,34)(w6,86)(c3,80)]
WRITE [(l1,32)(x8,78)(k0,95)]
DELETE [k6,i3,k0,i0,m1]
SHOW
WRITE [(s2,36)]
DELETE [n3,n7]
READ [e5]
WRITE [(r8,26)]
WRITE [(j4,70)(b7,91)]
WRITE [(u5,91)(c5,39)(p6,85)(v8,9)(u6,32)]
WRITE [(u1,94)(m5,44)(n0,85)(m2,9)(q9,20)]
WRITE [(i9,46)]
WRITE [(k9,48)]
READ [y0]
SHOW
WRITE [(y4,83)]
WRITE [(n2,35)(z9,46)(v4,52)(q8,61)(o9,85)]
READ [n2]
WRITE [(s5,79)]
WRITE [(h0,44)(h4,11)(z6,36)]
WRITE [(t3,11)(c4,57)(w8,87)(l6,34)(j9,23)]
READ [p6]
DELETE [y0]
WRITE [(o2,84)(q4,10)]
WRITE [(z5,33)(r6,7)(p4,76)(v5,96)]
DELETE [c1,r6,q9]
WRITE [(t1,30)]
WRITE [(w5,65)(y1,76)(v7,47)(h9,69)(r7,28)]
WRITE [(z8,47)(z1,62)(x9,100)(d9,41)(m6,19)]
WRITE [(r0,2)(k4,63)(n3,93)(h1,86)]
WRITE [(e4,30)(l9,57)]
READ [d9]
READ [w6]
READ [e0]
WRITE [(x6,6)(y7,64)(d5,64)]
READ [j5]
DELETE [m8,j3,z8]
WRITE [(m9,14)(z3,45)(v9,51)(t7,46)(i8,79)]
READ [j5]
READ [j4]
READ [m2]
DELETE [l6,h1,v8]
READ [n3]
WRITE [(l6,22)(s9,79)]
READ [n3]
WRITE [(b3,26)(g5,27)(h7,79)]
WRITE [(a6,5)(f5,38)(g2,31)(f2,29)(o4,0)]
WRITE [(k7,89)(k2,7)(q9,12)(y0,16)]
DELETE [e4,o4,c2,u5,r5]
DELETE [z4,n0]
WRITE [(d6,20)]
