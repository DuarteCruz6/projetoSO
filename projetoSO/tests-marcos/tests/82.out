(c3, 80)
(c1, 34)
(c2, 72)
(e0, 90)
(e5, 24)
(f8, 85)
(h8, 21)
(j3, 76)
(j0, 34)
(j5, 64)
(l1, 32)
(l2, 42)
(m8, 42)
(n4, 86)
(n3, 29)
(n7, 33)
(r5, 7)
(w6, 86)
(x8, 78)
(y0, 69)
(z4, 31)
[(e5,24)]
[(y0,69)]
(b7, 91)
(c5, 39)
(c3, 80)
(c1, 34)
(c2, 72)
(e0, 90)
(e5, 24)
(f8, 85)
(h8, 21)
(i9, 46)
(j4, 70)
(j3, 76)
(j0, 34)
(j5, 64)
(k9, 48)
(l1, 32)
(l2, 42)
(m2, 9)
(m5, 44)
(m8, 42)
(n0, 85)
(n4, 86)
(p6, 85)
(q9, 20)
(r8, 26)
(r5, 7)
(s2, 36)
(u1, 94)
(u6, 32)
(u5, 91)
(v8, 9)
(w6, 86)
(x8, 78)
(y0, 69)
(z4, 31)
[(n2,35)]
[(p6,85)]
[(d9,41)]
[(w6,86)]
[(e0,90)]
[(j5,64)]
[(j5,64)]
[(j4,70)]
[(m2,9)]
[(n3,93)]
[(n3,93)]
